src/System.Net.Security/src/System/IO/StreamAsyncHelper.cs

```diff
@@ -0,0 +1,193 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Diagnostics;
+using System.Diagnostics.Contracts;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace System.IO
+{
+    /// <summary>Provides support for implementing asynchronous operations on Streams.</summary>
+    internal sealed class StreamAsyncHelper
+    {
```

Comment:
Actually this was in Common\IO\StreamAsyncHelper and got removed by @Maxwe11 in 2732ffa58073e287fc68f5a1623cbf4c0700f44c just before I've finished this work.

Do you think I should move this back?